.github/workflows/build-and-release.yml

```diff
@@ -8,7 +8,7 @@ on:
 
 jobs:
   Test:
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-20.04
```

Comment:
please remind me why we need 20.04, is it for python 2.7?

Reply:
cause the build doesn't work with `ubuntu-latest` https://github.com/inspirehep/inspire-json-merger/actions/runs/4082131670/jobs/7036154967#step:4:7, if you know a better fix for that I can change it

Reply:
waiting to drop `next`